src/vstart.sh

```diff
@@ -1197,10 +1203,32 @@ start_cephexporter() {
 
 do_balance_cpu() {
     local osd=$1
+    local alienstore_idx=$(( osd + CEPH_NUM_OSD ))
+
+    local reactor_interval=${cpu_table[${osd}]}
+    if ! [ "${reactor_interval}" == "" ]; then
+        local cmd="$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_seastar_cpu_cores ${reactor_interval}"
+        echo $cmd
+        $cmd
+    else
+        echo "No cpu_table entry for osd $osd, setting crimson_seastar_num_reactors"
+        local cmd="$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_seastar_num_threads $crimson_smp"
+        echo $cmd
+        $cmd
+        return
+    fi
+
+
+    local alienstore_interval=${cpu_table[${alienstore_idx}]}
+    if [ ! "${alienstore_interval}" == "" ]; then
```

Comment:
`if [ "$objectstore" == "bluestore" ]; then`
Do we want to include this bit only for bluestore vstart clusters? same as we do later on for crimson_alien_num_threads later on

Reply:
discussed offline - setting alienstore_interval for no bluestore clusters will be non-harmful